test/e2e/service.go

```diff
@@ -330,7 +330,7 @@ var _ = Describe("Services", func() {
 
 		By("hitting the pod through the service's external load balancer")
 		var resp *http.Response
-		for t := time.Now(); time.Since(t) < time.Minute; time.Sleep(5 * time.Second) {
+		for t := time.Now(); time.Since(t) < podStartTimeout; time.Sleep(5 * time.Second) {
```

Comment:
waitForPodRuninng makes sure the pod is in the running state. Here we are waiting for it to be reachable through the external load balancer. It might be clearer if we used a different name for the timeout here.